src/components/chat/UserCard.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import * as S from '../../styles/chat/UserCardStyles';
+import { UserType } from '../../types/ChatType';
+
+interface IUserCardProps {
+  user: UserType;
+  selectedUser: UserType;
+  setSelectedUser: React.Dispatch<React.SetStateAction<UserType>>;
+}
+
+function UserCard({ user, selectedUser, setSelectedUser }: IUserCardProps) {
+  const handleSelectedUser = () => {
+    if (selectedUser.id === user.id) {
+      setSelectedUser({ id: '', name: '', picture: '' });
+      return;
+    }
+
+    setSelectedUser(user);
+  };
```

Comment:
요런 걸 useCallback으로 정의해봐도 좋겠죠~
```suggestion
  const handleSelectedUser = useCallback (() => {
    if (selectedUser.id === user.id) {
      setSelectedUser({ id: '', name: '', picture: '' });
      return;
    }

    setSelectedUser(user);
  }, [user]}
```